test/e2e/clusterctl_upgrade.go

```diff
@@ -263,6 +277,22 @@ func ClusterctlUpgradeSpec(ctx context.Context, inputGetter func() ClusterctlUpg
 			})
 
 			if !input.SkipCleanup {
+				if discovery.ServerSupportsVersion(managementClusterProxy.GetClientSet().DiscoveryClient, clusterv1.GroupVersion) == nil {
+					Byf("Deleting all clusters in Namespace: %s in management cluster: %s", testNamespace.Name, managementClusterName)
```

Comment:
```suggestion
					Byf("Deleting all clusters in namespace: %s in management cluster: %s", testNamespace.Name, managementClusterName)
```
nit (same below)

\+ golangci-lint finding